### DISCUSSION TITLE
Downstream DE analysis after DiaNN with Limma

### DISCUSSION THREAD

**Discussion by FrAoJm** [vdemichev/DiaNN#287] [Q&A] [answered]
Hi, 
I hope this is the correct place to ask this. I am quite new to proteomic analysis and I am in charge of performing a DE analysis from a DiaNN output (already normalised by DiaNN with "global normalization"). An I would like to have some advice and confirmation about the methods I used in order to get confidence in my results (any advice will be very helpful)
The full experiment consists of 2 types of similar tissues with 8 conditions each (and 3 to 6 biological replicates per condition). 
The original input has about 3000 proteins quantified and 3 technical replicates per sample so this is what I did:

1. Filter the proteins with more than 25% of missing values
2. Imputation with the minimal value (as I read in DiaNN Q/A) all the remaining missing values
3. I collapsed the tech replicates with the mean of the three values per protein and sample
4. The mean-collapsed data (proteins in rows and samples in columns) was log2-transformed
5. The previous data table was used as input for Limma DE analysis, including all the possible combinations (pairwise) of contrast in the model formula with the aim of generating individual contrasts and getting each pairwise comparison in individual tables. 

I am getting a very low number of significant DE proteins in most of the proteins and I am wondering if it is something of the experimental design (low biological replicates and/or low biological variance between conditions) or if the pipeline I am using is someway wrong. 

I will appreciate any "light" in this situation to be able to address the best solution, 

Best wishes

**Comment by vdemichev** [comment, marked as answer]
Hi, it's really difficult to say anything about the analysis without looking at the actual data in depth. So just some comments:
1. Filtering out proteins based on missing values is usually not necessary. But do filter based on Global.PG.Q.Value (if not using MBR) or Lib.PG.Q.Value (if using MBR).
2. DE analysis can be performed without imputation using limma. Whether or not imputaiton is beneficial - depends on the dataset.
3. Makes sense. Can also try median. Actually I would probably do this after the log2 transformation step.
4. Yes, we also log2 transform everything.

Low number of DE proteins: in case something is going wrong with limma, can always just do a simple thing and compare two conditions which you think will differ most using Mann-Whitney test. And then look what's significant and how the distribution of p-values looks.

One thing you can try: select only proteins which are very well quantified. One way to do this: select proteins that have been matched to, say, minimum 2 or 3 peptides at Global.Q.Value <= 0.01. Try making this filter stricted (more peptides) to reduce the number of proteins considered. Try doing GSEA - it does not require any significant hits.

Best,
Vadim

**Reply by Yi-Wo** [reply]
I'm new to this software and I'm trying to learn how to use it but I'm not sure in which output document I would be able to find the Lib.PG.Q.Value.  I tried looking in report.tsv and the report-lib.tsv. 

Thank you.

**Reply by vdemichev** [reply]
In the main report (report.tsv by default).
